README.md

```diff
@@ -1,24 +1,21 @@
 # betaflight-tx-lua-scripts
 
-**Important:**
-
-*Some changes in OpenTX 2.2.1 cause this version to have less RAM available for lua scripts than previous versions. The recommendation is for users wanting to update OpenTX on a Taranis X9D (and keep using the Betaflight TX lua scripts) is to use 2.3.1 or higher.*
-
 ## How to download the scripts
 
 Please visit the [releases page](https://github.com/betaflight/betaflight-tx-lua-scripts/releases) to download a zip file containing latest version.
 
 ## Firmware Considerations
 
-- Betaflight - As a best practice, it is recommended to use the most recent stable release of Betaflight to obtain the best possible results.
-- Crossfire - v2.11 or greater
-- FrSky - While most receivers work fine, it's recommended to update the XSR family of receivers to their most recent firmware version to correct any known bugs in SmartPort telemetry.
+- Betaflight - As a best practice, it is recommended to use the most recent stable release of Betaflight to obtain the best possible results;
+- OpenTX - 2.3 or newer;
```

Comment:
2.3.4 or newer
It's needed for protocol detection to work properly

Reply:
Fixed. Thanks.